build.gradle

```diff
@@ -20,6 +20,7 @@ jar {
   manifest {
     attributes 'Implementation-Title': 'Manifold Frontend', 'Implementation-Version': version, 'Main-Class': 'org.manifold.compiler.front.Main'
   }
+  from { configurations.antlr4.collect { it.isDirectory() ? it : zipTree(it) } }
```

Comment:
This includes the ANTLR4 runtime libraries in the produced jar `manifold-frontend-X.Y.Z.jar`, which has the effect of removing a runtime external dependency on these extra jars.